hw/bitstream/vivado/BUILD

```diff
@@ -23,11 +23,15 @@ _PREFIX = "../../../../../../../../../.."
 
 _FPGA_CW310_TESTROM = "//sw/device/lib/testing/test_rom:test_rom_fpga_cw310_scr_vmem"
 
-_FPGA_CW310_OTP_RMA = "//hw/ip/otp_ctrl/data:img_rma"
+_FPGA_CW340_TESTROM = _FPGA_CW310_TESTROM
```

Comment:
Do we need the `FPGA_` prefixes on these variables if they're contained to this file? If this is just the style, then that's fine.

Reply:
We don't, I think. I simply kept the form that others had started, but it can be changed.

Reply:
Okay, fine to leave as-is then, thanks

Reply:
Haha, too late! Already did.